DisCatSharp.ApplicationCommands/Attributes/ContextMenu/ContextMenuAttribute.cs

```diff
@@ -53,12 +53,18 @@ public sealed class ContextMenuAttribute : Attribute
 		/// </summary>
 		internal bool? DmPermission { get; set; }
 
+		/// <summary>
+		/// Gets whether this command is marked as NSFW
```

Comment:
Missing period

Reply:
The whole class is missing period ^^

Reply:
That's incorrect. Other classes do, this one doesn't.

Reply:
Oh fuck, well todo for generalo checkup

Reply:
Yeah should probably make it consistent across classes and maybe enforce it via CI or sth